Riot/Modules/Room/Views/BubbleCells/RoomTypingBubbleCell.swift

```diff
@@ -0,0 +1,125 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+
+@objcMembers
+class RoomTypingBubbleCell: UITableViewCell {
+    // MARK: - Constants
+    
+    static let cellIdentifier = "RoomTypingBubbleCell"
+    
+    // MARK: - Outlets
+    
+    @IBOutlet weak var additionalUsersLabel: UILabel?
```

Comment:
We could make outlets private and unwrap them safely, like:

`@IBOutlet private weak var additionalUsersLabel: UILabel!`